src/components/bcros/filing/addStaffFiling/modal/Form.vue

```diff
@@ -321,7 +322,28 @@ const handleSubmit = () => {
             </div>
             <div class="flex">
               <span class="font-bold mt-3 mr-5">Upload File</span>
+              <BcrosDocumentUpload
+                v-if="enableDocumentRecords"
+                v-model="staffNotation.courtOrderFile"
+                class="flex-grow"
+                :max-size="MAX_FILE_SIZE"
+                :page-size="PageSizeE.LETTER_PORTRAIT"
+                :disabled="submissionInProgress"
+                :upload-document-to-drs="uploadDocumentToDRS"
+                name="courtOrderFile"
+                data-cy="court-order-upload"
+                @set-error="(error) => {
+                  fileError = error
+                  validate()
+                }"
+                @clear-selection="() => {
+                  staffNotation.courtOrderFile = undefined
+                  fileError = undefined
+                  validate()
+                }"
+              />
```

Comment:
Aw, nuts. This is my fault...

Concurrently with this PR, Rajan has stripped out the Court Order filing from this staff filing modal, and it's the only one that supported file upload.

The Court Order filing is now implemented as a stand-alone filing page in Filings UI.

See:
- https://github.com/bcgov/business-dashboard-ui/pull/144
- https://github.com/bcgov/business-filings-ui/pull/714

Reply:
I'm not saying this PR can't be merged, but when you rebase, you will no longer have the above code to update.